registry/remote/example_test.go

```diff
@@ -352,25 +380,68 @@ func ExampleRepository_Fetch_manifestByDigest() {
 	}
 	// verify the fetched content
 	if descriptor.Size != int64(len(pulled)) || descriptor.Digest != digest.FromBytes(pulled) {
-		panic(err)
+		panic("wrong content")
 	}
 
 	fmt.Println(string(pulled))
 	// Output:
 	// Example manifest content
 }
 
-// ExampleRepository_FetchReference_manifestByTag gives example snippets for downloading a manifest by tag with only one API call.
-func ExampleRepository_FetchReference_manifestByTag() {
-	reg, err := remote.NewRegistry(host)
+// ExampleRepository_Fetch_artifactReferenceManifest gives an example of fetching
+// the reference manifests of a given manifest by using the Referrers API.
+func ExampleRepository_Fetch_artifactReferenceManifest() {
+	repo, err := remote.NewRepository(fmt.Sprintf("%s/%s", host, exampleRepositoryName))
 	if err != nil {
 		panic(err)
 	}
 	ctx := context.Background()
-	repo, err := reg.Repository(ctx, exampleRepositoryName)
+
+	// resolve a manifest by tag
+	tag := "latest"
+	descriptor, err := repo.Resolve(ctx, tag)
 	if err != nil {
 		panic(err)
 	}
+	// find its referrers by calling Referrers
+	// for each page of the returned value, do the following:
+	if err := repo.Referrers(ctx, descriptor, "", func(got []artifactspec.Descriptor) error {
```

Comment:
It's better to use `referrers` as the argument name

Reply:
changed.